dask/dataframe/core.py

```diff
@@ -3383,6 +3383,19 @@ def to_frame(self, index=True, name=None):
             else:
                 return self.map_partitions(M.to_frame, meta=self._meta.to_frame())
 
+    @insert_meta_param_description(pad=12)
+    @derived_from(pd.Series)
+    def map(self, arg, na_action=None, meta=no_default):
+        """
+        Note that this method clears any known divisions.
+
```

Comment:
```suggestion

If your mapping function is monotonically increasing then consider applying
the mapping to the old divisions and assigning the new divisions to the
output. 
```

Reply:
Thoughts on making this a keyword? Something like `is_monotonic`, and then

```python
if is_monotonic:
    result.divisions = tuple(pd.Series(self.divisions).map(arg, na_action=na_action))
else:
    result = result.clear_divisions()
return result
```

I'm having a hard time gauging what the right thing to do here, but I think in the interest of safety we should clear the divisions by default, but make it easy to maintain / propagate them.

Reply:
I think `is_monotonic` is reasonable.